src/os_crypto/sha1/sha1_op.c

```diff
@@ -93,8 +93,30 @@ int OS_SHA1_Str2(const char *str, ssize_t length, os_sha1 output)
     return (0);
 }
 
-// Get the hexadecimal result of a SHA-1 digest
+int OS_SHA1_strings(os_sha1 output, ...) {
+    unsigned char md[SHA_DIGEST_LENGTH];
+    size_t n;
+    SHA_CTX c;
+    SHA1_Init(&c);
 
+    va_list parameters;
+    char* parameter = NULL;
+    va_start(parameters, output);
+    while(parameter = va_arg(parameters, char*), parameter) {
```

Comment:
According to the style guide, please separate `while` and `(`.

Reply:
Done